app/views/layout.erb

```diff
@@ -15,12 +15,19 @@
 <body>
   <div class="container">
     <strong>Together, we can TODO it!</strong>
-    <a href="/users/new">Sign Up</a>
-    <a href="/sessions/new">Sign In</a>
+    <% if not session[:user_id] %>
```

Comment:
Having a `current_user` helper would be an improvement on looking directly at the session here.